builtin/credential/app-id/backend_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"testing"
 
-	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
+	logicaltest "github.com/hashicorp/vault/sdk/acctest"
```

Comment:
Since you're aliasing it as logicaltest, why not call it that?  I know I'm not the only one who's a bit uncomfortable with our misusage of the "acceptance test" terminology.  As further argument, consider that there's an `AcceptanceTest` field in `TestCase`, so when it's false the package name is misleading.

Reply:
Sorry for the duplicate comment, I guess the comment that I thought disappeared actually was submitted.

Reply:
Hey @ncabatoff - I moved/renamed the package from `sdk/acctest` to `sdk/testing/stepwise` as we discussed. As for changing the usage of import renaming across the existing tests, I'm leaning towards keeping them as is. Imagining that we'll add a future `Step` type akin to the current `TestStep`, I'm thinking that the current usage of `logical.TestStep` et. al. will be clearly distinguishable from future `stepwise.Step` usage. If I did remove the import renaming, the existing tests would be using `stepwise.TestStep` and then future tests would use `stepwise.Step`, which I worry could be confusing. Do you have thoughts on this?

Reply:
Not really, I don't have a preference for how you want to address migration to your new paradigm.  What you propose sounds reasonable.

Reply:
👍 thanks Nick. I'm going to leave things as-is for now then. I'm also going to point this PR at a long-running branch instead of master as well.